docs/theming/class-names-update-330.md

```diff
@@ -45,4 +45,8 @@ WC Blocks 3.3.0, introduced express payment methods in the Cart block. In order
 		<td><code>wc-block-components-express-checkout-payment-event-buttons</code></td>
 		<td><code>wc-block-components-express-payment__event-buttons</code></td>
 	</tr>
+	<tr>
+		<td><code>wc-block-components-checkout-form</code></td>
+		<td><code>wc-block-checkout__form</code></td>
```

Comment:
Ah I see this change is documented! 👌

Reply:
Yes, and moved it to a specific file for 3.4.0 in 0710e73a2e856f84384667451904f1e8e736f08d.